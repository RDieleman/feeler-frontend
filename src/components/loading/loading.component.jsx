import React from "react";
import "./loading.styles.css";

export const LoadingComponent = () =>{
    return(
        <div className="loading-container">
            <div className="lds-ring">
                <div></div>
                <div></div>
                <div></div>
                <div></div>
            </div>
        </div>
    )
}