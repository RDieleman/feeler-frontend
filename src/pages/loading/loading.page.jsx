import React, {Component} from "react";
import "./loading.styles.css";

/*
    Page that will be displayed when loading state is active.
    Page replaces the whole app component so resources can be safely loaded.
    Only contains the logo and title of the app.
 */

class LoadingPage extends Component{
    constructor(props) {
        super(props);

        this.state = {

        }
    }

    render() {
        return (
            <div >
                Loading...
            </div>
        );
    }
}

export default LoadingPage;