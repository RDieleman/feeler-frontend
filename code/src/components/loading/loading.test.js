import React from "react";
import {shallow} from "enzyme";
import {LoadingComponent} from "./loading.component";

it("Expect to render Loading component", () => {

    expect(shallow(
        <LoadingComponent

        />)).toMatchSnapshot();
});