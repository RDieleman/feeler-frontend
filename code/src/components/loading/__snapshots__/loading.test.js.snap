// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Loading component 1`] = `ShallowWrapper {}`;
